tests/common.go

```diff
@@ -1827,11 +1827,11 @@ func PerformSystemCheck() {
 					TimeBeforeRetry: 10 * time.Second,
 				})
 				if len(file) != 0 || err != nil {
-					log.Errorf("an error occurred, collecting bundle")
+					dash.Errorf("Core file was found on node %s, Core Path: %s", n.Name, file)
+					// Collect Support Bundle only once
 					CollectSupport()
-					dash.VerifySafely(file == "", true, fmt.Sprintf("Core generated on node %s, Core Path: %s", n.Name, file))
+					log.Fatalf("Core generated, please check logs for more details")
```

Comment:
you have to add `break` here to get out from loop

Reply:
no need @mborodin-px , once the log.Fatal is called it will fail the test and return